composer.json

```diff
@@ -195,6 +196,9 @@
             },
             "drupal/easy_install": {
                 "Handle missing core version": "https://www.drupal.org/files/issues/2019-12-19/easy_install_core_version-3101883-0.patch"
+            },
+            "drupal/google_analytics_counter": {
+                "Add API query tweak options": "https://gist.githubusercontent.com/omahm/02ac3b7138c4d0518bd954721b74b763/raw/8d7ffa08b21359c7bf11165cdef1fb7e98483774/gac_query_options.patch"
```

Comment:
If this is re-usable, might be worth adding a new drupal.org issue + attach the patch there?

Reply:
Yup, added it as a feature request yesterday https://www.drupal.org/project/google_analytics_counter/issues/3110812